src/main/java/uk/ac/ic/wlgitbridge/bridge/db/postgres/PostgresDBStore.java

```diff
@@ -0,0 +1,298 @@
+package uk.ac.ic.wlgitbridge.bridge.db.postgres;
+
+import uk.ac.ic.wlgitbridge.bridge.db.DBInitException;
+import uk.ac.ic.wlgitbridge.bridge.db.DBStore;
+import uk.ac.ic.wlgitbridge.bridge.db.ProjectState;
+import uk.ac.ic.wlgitbridge.util.Log;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PostgresDBStore implements DBStore {
+
+  private final Connection connection;
+
+  public PostgresDBStore(String connectionString, String user, String pass) {
+    Log.info("Initialize PostgresDBStore");
+    try {
+      Class.forName("org.postgresql.Driver");
+      connection = DriverManager
+        .getConnection(connectionString, user, pass);
+    } catch (Exception e) {
+      Log.error("Error connecting to Postgres: {}", e.getMessage());
+      throw new DBInitException(e);
+    }
+  }
+
+  public void dangerousResetDatabaseDontDoIt() {
```

Comment:
I don't see this method in the base class or the SQLite implementation. Can we drop it?

Reply:
We need it to make the acceptance tests work. They need to reset the database before each test case runs. This is not an issue for testing with Sqlite, because each test gets a new directory with a new Sqlite database file.

We could add it to the interface and make it work on the Sqlite instance too?